ingest-app/src/cmr/ingest/runner.clj

```diff
@@ -10,5 +10,5 @@
   [& args]
   (let [system (system/start (system/create-system))]
     (cfg/check-env-vars)
-    (events/stop-on-exit-hook (:instance-name system) #(system/stop system))
+    (events/stop-on-exit-hook #(system/stop system))
```

Comment:
so the system name was to print out which task in AWS closed successfully. I was thinking that we could track how many started and never report a clean shutdown.

Reply:
Good point, we missed the purpose/the supporter. Reverting change